### PR TITLE
Link hover styling v2

### PR DESCRIPTION
**Pull request by karmatosed**
v3 PR for #240 fix

### CODE REVIEW

**Comment by josephfusco** [comment]
Same comments I made in #326,

> Multiple comma-separated values for one property should be separated by either a space or a newline, including within rgba().

rgba values should be fixed to follow [WP CSS coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/css/#values).

File: style.css [hunk 1]
```diff
@@ -783,9 +782,8 @@ a:active {
 .site-footer .widget-area a,
 .posts-navigation a,
 .widget_authors a strong {
-	border-bottom: 2px solid #767676;
-	-webkit-transition: background-color 0.2s ease-in-out, border-color 0.2s ease-in-out, color 0.3s ease-in-out;
-	transition: background-color 0.2s ease-in-out, border-color 0.2s ease-in-out, color 0.3s ease-in-out;
+	box-shadow: inset 0 -1px 0 rgba( 15, 15, 15, 1.0);
```
Comment (ntwb) [inline style.css:785]: Remove the space after the opening brace and remove the trailing zero i.e this: `rgba(15, 15, 15, 1);`

File: style.css [hunk 2]
```diff
@@ -814,9 +812,9 @@ a .nav-title,
 .widget ul li a:visited,
 .site-footer .widget-area ul li a,
 .site-footer .widget-area ul li a:visited {
-	border: 0;
-	-webkit-transition: background-color 0.2s ease-in-out, border-color 0.2s ease-in-out, color 0.3s ease-in-out;
-	transition: background-color 0.2s ease-in-out, border-color 0.2s ease-in-out, color 0.3s ease-in-out;
+	box-shadow: inset 0 -1px 0 rgba( 15, 15, 15, 1.0);
```
Comment (ntwb) [inline style.css:815]: Same as above, `rgba(15, 15, 15, 1)`, not `rgba( 15, 15, 15, 1.0)`

File: style.css [hunk 3]
```diff
@@ -841,9 +839,8 @@ a .nav-title,
 .widget_authors a:hover strong,
 .project-terms a:focus,
 .project-terms a:hover {
-	border-bottom-color: transparent;
-	background-color: #767676;
-	color: #fff;
+	color: #000;
+	box-shadow: inset 0 0 0 rgba( 0, 0, 0, 0), 0 3px 0 rgba(0,0,0, 1);
```
Comment (ntwb) [inline style.css:843]: No space after opening parenthesis, space after comma
 i.e `box-shadow: inset 0 0 0 rgba(0, 0, 0, 0), 0 3px 0 rgba(0, 0, 0, 1);`

File: style.css [hunk 4]
```diff
@@ -876,8 +873,8 @@ a:hover .nav-title,
 .widget .widget-title a:hover,
 .widget ul li a:focus,
 .widget ul li a:hover {
-	background-color: #767676;
-	color: #fff;
+	color: #000;
+	box-shadow: inset 0 0 0 rgba( 0, 0, 0, 0), 0 3px 0 rgba(0,0,0, 1);
```
Comment (ntwb) [inline style.css:877]: Same as above: `box-shadow: inset 0 0 0 rgba(0, 0, 0, 0), 0 3px 0 rgba(0, 0, 0, 1);`

**Review by ntwb — CHANGES_REQUESTED** [review]
Added some CSS coding standards notes

**Comment by melchoyce** [comment]
Seeing underlines on every link now:

screen shot 2016-10-17 at 3 43 02 pm
screen shot 2016-10-17 at 3 44 19 pm

**Comment by laurelfulford** [comment]
I created a new PR - #428 - based on @karmatosed's work. It addresses the formatting changes, and limits where the hovers appear by default. Should line up with where they appeared originally.

@melchoyce - let me know if anything needs changing!

**Comment by josephfusco** [comment]
Closing this in favor of #428. 👍

**Pull request closed without merging**